gaia-dialog-select.js

```diff
@@ -26,6 +26,20 @@ module.exports = component.register('gaia-dialog-select', {
     this.els.list.addEventListener('click', this.onListClick.bind(this));
     this.els.submit.addEventListener('click', this.close.bind(this));
     this.els.cancel.addEventListener('click', this.close.bind(this));
+
+    setTimeout(() => this.makeAccessible());
+  },
+
+  makeAccessible() {
+    this.els.list.setAttribute('role', 'listbox');
+    if (this.multiple) {
+      this.els.list.setAttribute('aria-multiselectable', true);
+    }
```

Comment:
eslint fails if i one-line it and remove {}. If {} are left in place, line's too long too..